src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsPatch.kt

```diff
@@ -1,39 +1,42 @@
-package app.revanced.patches.youtube.misc.settings.bytecode.patch
+package app.revanced.patches.youtube.misc.settings
 
 import app.revanced.extensions.exception
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
 import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
 import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
 import app.revanced.patcher.extensions.InstructionExtensions.replaceInstruction
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotation.Patch
 import app.revanced.patches.shared.settings.preference.impl.Preference
 import app.revanced.patches.shared.settings.util.AbstractPreferenceScreen
-import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
-import app.revanced.patches.youtube.misc.settings.bytecode.fingerprints.LicenseActivityFingerprint
-import app.revanced.patches.youtube.misc.settings.bytecode.fingerprints.SetThemeFingerprint
-import app.revanced.patches.youtube.misc.settings.resource.patch.SettingsResourcePatch
+import app.revanced.patches.youtube.misc.integrations.IntegrationsPatch
+import app.revanced.patches.youtube.misc.settings.fingerprints.LicenseActivityFingerprint
+import app.revanced.patches.youtube.misc.settings.fingerprints.SetThemeFingerprint
 import com.android.tools.smali.dexlib2.Opcode
 import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
 import com.android.tools.smali.dexlib2.util.MethodUtil
 import java.io.Closeable
 
-@DependsOn([IntegrationsPatch::class, SettingsResourcePatch::class])
-@Name("Settings")
-@Description("Adds settings for ReVanced to YouTube.")
-class SettingsPatch : BytecodePatch(
-    listOf(LicenseActivityFingerprint, SetThemeFingerprint)
+@Patch(
+    description = "Adds settings for ReVanced to YouTube.",
+    dependencies = [IntegrationsPatch::class, SettingsResourcePatch::class]
+)
```

Comment:
❓ : Missing patch name

```suggestion
@Patch(
    name = "Settings",
    description = "Adds settings for ReVanced to YouTube.",
    dependencies = [IntegrationsPatch::class, SettingsResourcePatch::class]
)
```